packages/components/src/card/stories/footer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import { boolean, text } from '@storybook/addon-knobs';
+/* eslint-enable import/no-extraneous-dependencies */
+
+/**
+ * Internal dependencies
+ */
+import Card from '../index';
+import CardFooter from '../footer';
+import { getCardStoryProps } from './__utils';
+
+export default { title: 'Card/Footer', component: CardFooter };
```

Comment:
Adding the `/` automatically creates sub-stories within Storybook :). This allows for better organization, like with the `<CardMedia />` component. It's really neat!

![Screen Shot 2019-10-17 at 11 13 51 AM](https://user-images.githubusercontent.com/2322354/67022472-47805500-f0cf-11e9-813c-b717fed2a133.png)

Reply:
Nice, I see they only support `|` for collections. Given that the name of the component is still prefixed with `Card`, should we title it `Card/CardFooter`? I'm not convinced it would read well. Your call :)